locales/de_DE/LC_MESSAGES/duckduckgo.po

```diff
@@ -3085,39 +3085,39 @@ msgid "Select %sEdit Search Engines...%sin the dropdown"
 msgstr "Wähle %sSuchmaschinen bearbeiten...%s  im Dropdownmenü"
 
 msgid "Select %sManage add-ons%s from the dropdown menu"
-msgstr "Wählen sie %sAddons%s aus dem Dropdownmenü"
+msgstr "Wähle %sAddons%s aus dem Dropdownmenü"
 
 msgid "Select %sOpera > Preferences%s (on Mac) or %sMenu > Settings%s (on Windows)"
 msgstr "Wählen Sie %sOpera > Einstellungen%s (auf Mac) oder %sMenü > Einstellungen%s (auf Windows)"
 
 #. http://i.imgur.com/WU0BbCB.png
 msgid "Select %sOpera > Preferences%s (on Mac) or %sOpera > Options%s (on Windows)"
-msgstr "%sOpera → Einstellungen%s (beim Mac) or %sOpera → Optionen%s (in Windows)"
+msgstr "%sOpera → Einstellungen%s (beim Mac) or %sOpera → Optionen%s (in Windows) wählen"
 
 msgid "Select %sPreferences%s."
 msgstr "%sEinstellungen%s auswählen."
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSearch engine%s, then %sOthers...%s"
-msgstr ""
+msgstr "Wähle %sSuchmaschine%s, dann  %sWeitere...%s"
 
 msgid "Select %sSettings%s from the drop-down menu."
 msgstr "%sEinstellungen%s in der Liste auswählen."
 
 msgid "Select %sSettings%s"
-msgstr "%sEinstellungen%s auswählen "
+msgstr "Wähle %sEinstellungen%s"
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSettings%s, then %sAdvanced settings%s"
-msgstr ""
+msgstr "Wähle %sEinstellungen%s, dann %Erweiterte EEinstellungen%s"
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSettings%s, then %sDefault search engine%s"
-msgstr ""
+msgstr "Wähle %sEinstellungen%s, dann %sStandard-Suchmaschine%s"
 
 msgctxt "mobile homepage banner"
 msgid "Select %sSettings%s, then %sSearch Engine%s"
-msgstr "%sEinstellungen%s und dann %sSuchmaschine%s auswählen"
+msgstr "Wähle %sEinstellungen%, dann %sSuchmaschine%s"
```

Comment:
Many thanks @oliver-gramberg!

Just one minor issue with this placeholder...

```suggestion
msgstr "Wähle %sEinstellungen%s, dann %sSuchmaschine%s"
```

Reply:
Oops. Sure!